wpeverest/everest-forms
# Fix - Wrong instruction on captcha

### All Submissions:






### Changes proposed in this Pull Request:



Previously, Wrong instruction on captcha. This PR solve that issue.

### How to test the changes in this Pull Request:

1. Click on any captcha fields you will see correct instruction (Everest Forms>Settings>Captcha>reCaptcha.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Wrong instruction on captcha
